docs/install.html

```diff
@@ -39,9 +40,9 @@ <h1>Install</h1>
 				<li><a href="#linux">Linux</a></li>
 			</ul>
 
-			<h2 id="source">Source</h2>
-			
-			<p class="desc">The source code for Night can be found on the <a href="github.com/DynamicSquid/night">GitHub page</a>.</p>
+			<h2 id="source" class="heading">Source</h2>
```

Comment:
I think `h2` is the only `h2` in this HTML page, so we don't need a class for it

Reply:
yeah, but unfortunately if you remove that class the h2 becomes black in color and has the default font

Reply:
gotcha, thanks for the bug! I fixed it by making `h2` global